media-proxy/include/mesh/conn.h

```diff
@@ -67,8 +67,14 @@ enum class Result {
     error_bad_argument,
     error_out_of_memory,
     error_general_failure,
+    error_already_initialized,
+    error_initialization_failed,
+    error_memory_registration_failed,
+    error_thread_creation_failed,
+    error_no_buffer,
+    error_timeout,
```

Comment:
This should go after `error_context_cancelled`

Reply:
fixed in https://github.com/OpenVisualCloud/Media-Communications-Mesh/pull/286/commits/794708adf96f927afbd18cdc93def4203541ba83